pkg/process/config/config.go

```diff
@@ -281,15 +280,6 @@ func loadEnvVariables() {
 			config.Datadog.Set(variable.cfg, v)
 		}
 	}
-
-	if v := os.Getenv("DD_ORCHESTRATOR_ADDITIONAL_ENDPOINTS"); v != "" {
```

Comment:
What is this being replaced by? I think we still need to support this env var

Reply:
Oh wait, yeah thanks for calling this out @just-chillin it was not part of our owned files 😬 indeed this is still actively used (both in the cluster-agent and process-agent)

Reply:
Why would you want to remove this bit?

Reply:
The custom unmarshaling isn't needed. `config.Datadog.GetStringMapStringSlice` will handle that format.

Reply:
This config is still read here: https://github.com/DataDog/datadog-agent/blob/1d75034f1cad5d3a853183567796feea97aa68ce/pkg/orchestrator/config/config.go#L122

Reply:
make sense 👍